dashboard_test.cfg

```diff
@@ -1,5 +1,4 @@
 [dashboard]
 TEST_DIR=./zeeguu_api/tests/
-LOG_DIR=./
 N=5
-SUBMIT_RESULTS_URL=https://zeeguu.unibe.ch/api/dashboard/submit-test-results
+SUBMIT_RESULTS_URL=http://143.176.113.105:5000/dashboard/submit-test-results
```

Comment:
@kloostert -- is this correct?

Reply:
No this is the IP of a test server on which I deployed the dashboard for testing it out a bit, but the second commit in this PR actually changes the url back to the Zeeguu dashboard.